django_prices/forms.py

```diff
@@ -22,42 +28,43 @@ def __init__(
         *args,
         **kwargs
     ):
-        self.currency = currency
+        self.default_currency = default_currency
```

Comment:
Why do we need to have a default currency?

Reply:
You're right, it is not necessary; I've removed it.